nucypher/blockchain/eth/sol/compile.py

```diff
@@ -26,12 +26,7 @@
 import itertools
 import shutil
 
-try:
-    from solc import install_solc, compile_files
-    from solc.exceptions import SolcError
-except ImportError:
-    # TODO: Issue #461 and #758 & PR #1480 - Include precompiled ABI; Do not use py-solc in standard installation
-    pass
+from nucypher.blockchain.eth.sol.__conf__ import SOLIDITY_COMPILER_VERSION
```

Comment:
If you want to preserve the importable from the namespace `eth` - you can move this line into `__init__.py` and instead have just `from nucypher.blockchain.eth import SOLIDITY_COMPILER_VERSION`

Reply:
good advice, thanks
✔️